src/test/java/org/jenkinsci/plugins/mesos/integration/MesosJenkinsAgentLifecycleTest.java

```diff
@@ -50,7 +51,26 @@ public void testAgentLifecycle(TestUtils.JenkinsRule j) throws Exception {
 
     final String name = "jenkins-lifecycle";
     final String idleMin = "1";
-    final MesosAgentSpecTemplate spec = new MesosAgentSpecTemplate(name, Mode.EXCLUSIVE);
+    LabelAtom label = new LabelAtom("label");
+    final MesosAgentSpecTemplate spec =
```

Comment:
Since this is repeated multiple times, we can have a builder/fixture for this ?!

Reply:
added to TODO to refactor tests